src/libratbag-data.c

```diff
@@ -269,6 +283,98 @@ init_data_steelseries(struct ratbag *ratbag,
 	}
 }
 
+static void
+init_data_asus(struct ratbag *ratbag,
+		GKeyFile *keyfile,
+		struct ratbag_device_data *data)
+{
+	const char *group = "Driver/asus";
+	int num, i;
+	GError *error = NULL;
+	_cleanup_(freep) char *str = NULL;
```

Comment:
Analogous to `strv`: make this a variable for a concrete thing to prevent mistakes from happening in the future.